.gitignore

```diff
@@ -9,3 +9,6 @@ venv
 *.pyc
 *.swp
 .coverage
+
+#mac
+.DS_Store
```

Comment:
I would suggest you to move this to the "# Folders" section

Reply:
done